eng/Versions.props

```diff
@@ -12,6 +12,7 @@
   </PropertyGroup>
   <!-- Production Dependencies -->
   <PropertyGroup>
+    <MicrosoftNETCoreAppRefPackageVersion>3.0.0-preview7-27826-20</MicrosoftNETCoreAppRefPackageVersion>
```

Comment:
I will change this to MicrosoftNETCoreAppPackageVersion to match other repos and CoherentParent elsewhere. But this is debt if we want to remove Microsoft.NETCore.App. It's a concern that spans many repos though, so I'll separate it.